lib/web_ui/dev/driver_version.yaml

```diff
@@ -1,4 +1,10 @@
-
+## Version to be picked by this PR. The LUCI will have multiple cipd packages
+## insalled in the recipe at the same time.
+## This mapping will be used to switch between them.
+preferred_version:
+  ## Make sure the major version of the binary in `browser_driver.yaml` is the
```

Comment:
`browser_driver.yaml` - there no such file. If this refers to `browser_lock.yaml`, then I'm not sure what "major version of the binary" means.

Reply:
yes it's a typo thanks for noticing.

Major version means "84" for Chrome version "84.0.13". These were referred as version at some point and you asked me to make it major version :) 

Major version is more relevant to integration tests, since tests require a driver in the same major version to be running. They still run if it's a driver for a different minor version.

Reply:
Changed the comment. I added a description for major version. I also added documentation to README.